R/series-to_r_vector.R

```diff
@@ -216,28 +216,40 @@ series__to_r_vector <- function(
     if (identical(int64, "integer64")) {
       if (!is_bit64_installed()) {
         abort(
-          "If the `int64` argument is set to 'integer64', the `bit64` package must be installed."
+          c(
+            "The `bit64` package is not installed.",
+            `*` = 'If `int64 = "integer64"`, the `bit64` package is must be installed.'
```

Comment:
```suggestion
            `*` = 'If `int64 = "integer64"`, the `bit64` package must be installed.'
```
(not sure if this will be overwritten by #323)